### PR TITLE
Upgrade to next 13.1

### LINKED ISSUE #6264: [CAL-675] upgrade to next 13.1
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-675](https://linear.app/calcom/issue/CAL-675/upgrade-to-next-131)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
waiting for this to be fixed:

CleanShot 2023-01-04 at 12 56 00@2x

### CODE REVIEW

**Review by zomars — APPROVED** [review]
- Made some type fixes
- [Applied necessary code mods for Next 13](https://nextjs.org/docs/advanced-features/codemods#)
- Tests are passing
- Type checks are passing

File: apps/web/next.config.js [hunk 1]
```diff
@@ -89,6 +89,12 @@ const nextConfig = {
   eslint: {
     ignoreDuringBuilds: !!process.env.CI,
   },
+  // TODO: We need to have all components in `@calcom/ui/components` in order to use this
```
Comment (zomars) [inline apps/web/next.config.js:92]: cc @PeerRich

File: apps/web/components/NavTabs.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ const NavTabs: FC<NavTabProps> = ({ tabs, linkProps, ...props }) => {
           const className = tab.className || "";
           return (
             <Component key={tab.name}>
-              <Link key={tab.name} href={href} {...linkProps}>
+              <Link key={tab.name} href={href} {...linkProps} legacyBehavior>
```
Comment (PeerRich) [inline apps/web/components/NavTabs.tsx:65]: do we want to use legacyBehavior or just refactor and remove the `<a>`?

File: apps/web/next.config.js (hunk 1 above)
Comment (PeerRich) [reply apps/web/next.config.js:92]: cc @JeroenReumkens

Comment (sean-brydon) [reply apps/web/next.config.js:92]: We're getting there!

**Comment by PeerRich** [comment]
tested and works!

File: apps/web/components/NavTabs.tsx (hunk 2 above)
Comment (zomars) [reply apps/web/components/NavTabs.tsx:65]: Depends on the use case

**Pull request merged**